cr17/msg/scoopControl.msg

```diff
@@ -0,0 +1,3 @@
+float64 arm
```

Comment:
This variable needs a more meaningful name to indicate it is an angular measurement.